cli.js

```diff
@@ -38,11 +38,11 @@ var argc = process.argv.length;
 if (argc < 3){
 	help();
 } else if (argc === 3){
-	movieArt(movie, null, null, cb);
+	movieArt(movie, null, null, 'tv', cb);
 } else if (argc === 4 && !isNaN(parseFloat(process.argv[3])) && isFinite(process.argv[3])){
 	movieArt(movie, process.argv[3], null, cb);
 } else if (argc === 4){
 	movieArt(movie, null, process.argv[3], cb);
 } else {
 	movieArt(movie, process.argv[3], process.argv[4], cb);
-}
\ No newline at end of file
+}
```

Comment:
There are a few problems with this logic, I'll shoot another message after work. (Thank you for the contribution!!!)